docs/reference/editor-settings.md

```diff
@@ -131,3 +131,34 @@ The extension uses the first executable that exists. This setting takes preceden
   "ty.path": ["/home/user/.local/bin/ty"]
 }
 ```
+
+## Neovim
```

Comment:
This section should actually be present under the https://github.com/astral-sh/ty/blob/main/docs/README.md#other-editors section instead. This document is only to provide reference for all editor settings.

Reply:
done